### DISCUSSION TITLE
manger.dois.update.success or manager.dois.update.success ?

### DISCUSSION THREAD

**Discussion by DariuszLis** [pkp/pkp-lib#8959] [Proposals]
Is the missing "a" in "manger.dois.update.success" (line 179 of pkp-lib/classes/components/listPanels/PKPDoiListPanel.php) a bug? Shouldn't it be "man**a**ger.dois.update.success"?

**Comment by asmecher** [comment]
@ewhanson, would you mind resolving this at some point? It doesn't look like a functional problem, as the typo is replicated throughout consistently.

**Reply by DariuszLis** [reply]
Sorry for my lack of precision. I just meant a typo. I thought that changing "manger" to "manager" in the pkp-lib/classes/components/listPanels/PKPDoiListPanel.php file and in *.po files - because "manger.dois.update.success" is used only in these files - should remove the problem.

**Reply by asmecher** [reply]
I see it also in `lib/ui-library/src/components/ListPanel/doi/DoiListPanel.vue` (which is compiled into JS when the .tar.gz package is built).

**Reply by DariuszLis** [reply]
I checked and you are right. I have searched all PKP repositories and it seems that there are no other places where `manger.dois.update.success` appears. So, since we have identified all places of occurrence (`pkp-lib/classes/components/listPanels/PKPDoiListPanel.php`, `ui-library/src/components/ListPanel/doi/DoiListPanel.vue` and `pkp-lib/locale/*/manager.po`), then do you think you can change `manger.dois.update.success` to `manager.dois.update.success` in them?

**Comment by jonasraoni** [comment]
I've fixed the typos, so I'm closing this discussion.
